templates/api/monitor_api_credential.html

```diff
@@ -0,0 +1,75 @@
+{% extends "accounts/_section.html" %}
+
+{% block head %}
+{{ block.super }}
+<style type="text/css">
+  .axis path,.axis line {
+    fill: none;
+    stroke: #b6b6b6;
+    shape-rendering: crispEdges;
+  }
+  g.legend { 
+    font: 10px sans-serif;
+  }
+  .legend-item{  
+    cursor: pointer;
+  }
+  .api-usage{
+    min-height: 200px;
+  }
+</style>
+{% endblock head %}
+
+{% block title %}
+    Monitor
+{% endblock title %}
+
+{% block content %}
+
+    <h2>Api usage</h2>
+    <h3>Api usage by day on last two weeks</h3>
+    <div class='api-usage'>
+    </div>
+
+    <script>
+      var queriesApiUrl = '{% url 'monitor-api-stats-ajax' client.client_id %}';
+      var dayLimit = {{ limit }};
+      $.get(queriesApiUrl, function(d){
+        var usage = [];
+        var limit = [];
+        for (var key in d.results) {
+          if (d.results.hasOwnProperty(key)) {
+            usage.push({day: parseInt(key)*1000, count: d.results[key]});
+            limit.push({day: parseInt(key)*1000, count: dayLimit});
+          }
+        }
+        if (usage.length > 0) {
+          displayCharts('.api-usage', [usage, limit], {
+              yText: 'Request',
+              attrX: 'day',
+              attrY: 'count',
+              timeFormat: "%a %d",
+              tickEvery: d3.timeDay.every(1),
+              legendData: [{color: 'crimson', name: 'requests'}, {color: 'black', name: 'rate limit'}]
+            }, {});
+        } else {
+          $('.api-usage').html('<p>There are no requests in the given period.</p>')
+        }
+      });
+
+    </script>
+    <script src="https://d3js.org/d3.v4.min.js" type="text/javascript"></script>
```

Comment:
It says you can use it here: https://d3js.org/

> To link directly to the latest release, copy this snippet: 
> ...



I prefer to have an external dependency if it's reliable since will be faster than FS (because of all the problems we have regarding IO), and it's probable that users have it cached.

We could also use: https://cdnjs.com/libraries/d3